decidim-admin/app/views/decidim/admin/newsletters/show.html.erb

```diff
@@ -16,9 +16,10 @@
 </div>
 <div class="button--double form-general-submit">
   <% if allowed_to?(:update, :newsletter, newsletter: @newsletter) %>
-    <% if  !@newsletter.sent? %>
+    <% unless @newsletter.sent? %>
```

Comment:
```suggestion
```
This shouldn't be here if we don't restrict the sending once the newsletter has been sent

Reply:
Done. Thanks @microstudi  :)